packages/angular-workspace/package.json

```
@@ -20,7 +20,11 @@
     "pack:application": "cd dist/example-client-app && npm pack",
     "performance": "lhci autorun",
     "watch": "ng build --watch --configuration development",
+    "tdd:nimble": "ng test @ni/nimble-angular --browsers=Chrome",
+    "tdd:spright": "ng test @ni/spright-angular --browsers=Chrome",
     "test": "ng test --watch=false",
+    "test:nimble": "ng test @ni/nimble-angular --watch=false",
+    "test:spright": "ng test @ni/spright-angular --watch=false",
```

It's not written down anywhere but the convention trying to follow in nimble is that script names with `:` are sub commands of other scripts. As written I'd expect something like `"tdd":"npm run tdd:nimble && npm run tdd:spright"`

Since these are separate top-level commands I'd expect:
```suggestion
    "tdd-nimble": "ng test @ni/nimble-angular --browsers=Chrome",
    "tdd-spright": "ng test @ni/spright-angular --browsers=Chrome",
    "test": "ng test --watch=false",
    "test-nimble": "ng test @ni/nimble-angular --watch=false",
    "test-spright": "ng test @ni/spright-angular --watch=false",
```